frontend/src/pages/modelRegistrySettings/CreateMRSecureDBSection.tsx

```diff
@@ -25,22 +27,25 @@ interface CreateMRSecureDBSectionProps {
   secureDBInfo: SecureDBInfo;
   modelRegistryNamespace: string;
   nameDesc: { name: string };
-  existingCertKeys: string[];
-  existingCertConfigMaps: string[];
-  existingCertSecrets: string[];
+  existingCertConfigMaps: ConfigSecretItem[];
+  existingCertSecrets: ConfigSecretItem[];
   setSecureDBInfo: (info: SecureDBInfo) => void;
 }
 
 export const CreateMRSecureDBSection: React.FC<CreateMRSecureDBSectionProps> = ({
   secureDBInfo,
   modelRegistryNamespace,
   nameDesc,
-  existingCertKeys,
   existingCertConfigMaps,
   existingCertSecrets,
   setSecureDBInfo,
 }) => {
-  const [searchValue, setSearchValue] = useState('');
+  const [configSecretName, setConfigSecretName] = useState('');
+  const [key, setKey] = useState('');
```

Comment:
Same here, this is the search input for the key and not the selected key so maybe we should rename it